cpu/stm32f0/periph/gpio.c

```diff
@@ -195,7 +202,7 @@ void isr_exti(void)
     for (size_t i = 0; i < EXTI_NUMOF; i++) {
```

Comment:
Before the for loop, you would need something like this: `uint32 isr = (EXTI->PR & EXTI->IMR)` and test `if (isr & (1 << i)) {` because the line can be configured, but that interrupt itself disabled :)